docs/source/documentation.rst

```diff
@@ -433,6 +433,16 @@ If the segmentation with AxonDeepSeg fails or does not give optimal results, you
 * Manually correct the axon mask (as explained in Option 1).
 * Use the `AxonSeg <https://github.com/neuropoly/axonseg>`_ software to segment the myelin from the axon mask. In order to do this, install AxonSeg, and then follow the instructions in part 5 of the `as_tutorial guideline <https://github.com/neuropoly/axonseg/blob/master/as_tutorial.m>`_.
 
+**Installing FSLeyes**
+ 
+* Install the FSLeyes_integration branch of AxonDeepSeg inside a conda environment
+* Install Wxpython using conda install -c anaconda wxpython
+* Install FSLeyes using pip install fsleyes
```

Comment:
This doesn't work for me on Mac. I get the following error when trying to run `fsleyes`:

```

(ads_182) mathieuboudreau@brassens:~/neuropoly/pr_review/axondeepseg/182$ fsleyes
This program needs access to the screen. Please run with a
Framework build of python, and only when you are logged in
on the main display of your Mac.

```